instrumentation/sidekiq/README.md

```diff
@@ -0,0 +1,53 @@
+# OpenTelemetry Sidekiq Instrumentation
+
+The Sidekiq instrumentation is a community-maintained instrumentation for the [Sidekiq][sidekiq-home] Ruby jobs system.
+
+## How do I get started?
+
+Install the gem using:
+
+```
+gem install opentelemetry-instrumentation-sidekiq
+```
+
+Or, if you use [bundler][bundler-home], include `opentelemetry-instrumentation-sidekiq` in your `Gemfile`.
+
+## Usage
+
+To use the instrumentation, call `use` with the name of the instrumentation:
+
+```ruby
+OpenTelemetry::SDK.configure do |c|
+  c.use 'OpenTelemetry::Instrumentation::Sidekiq'
+end
+```
+
+Alternatively, you can also call `use_all` to install all the available instrumentation.
+
+```ruby
+OpenTelemetry::SDK.configure do |c|
+  c.use_all
+end
+```
+## Examples
+
+Example usage can be seen in the `./example/sidekiq.rb` file [here](https://github.com/open-telemetry/opentelemetry-ruby/blob/master/instrumentation/sidekiq/example/sidekiq.rb)
```

Comment:
@mwear is it okay to add a link to the actual file here or is that considered bad practice considering it might become dead if there is ever a code restructure?

Reply:
I think it's ok fwiw, will defer to @mwear on any formatting prefs. nice work here @AzfaarQureshi

Reply:
👍 this is fine - if we restructure the code, we'll also fix the documentation.